src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/lookups.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Use lookup table data in queries
+tags:
+  - Query your data
+  - 'NRQL: New Relic Query Language'
+  - NRQL query tutorials
+metaDescription: In New Relic, how to query data added via CSV lookup tables.
+redirects:
+---
+
+import nrqlExampleStatusCodes from 'images/nrql_screenshot-crop_example-status-codes.webp'
+
+import nrqlLookupQueryTranslateItemIds from 'images/nrql_screenshot-crop_lookup-query-translate-item-ids.webp'
+
+import nrqlLookupQueryGeoip from 'images/nrql_screenshot-crop_lookup-query-geoip.webp'
+
+When you upload CSV-format lookup tables, you can use the `lookup()` function to access that data in your NRQL queries. 
+
+You can use lookups to make your telemetry data easier to understand, group data in custom ways, and include more information that wasn't available at the time for previously reported data. 
+
+## Upload a lookup table [#upload-table]
+
+For how to upload tables, see [Lookup tables](/docs/logs/ui-data/lookup-tables-ui). 
+
+## Technical details [#limitations]
```

Comment:
What do you think about putting this sort-of negative section after the query syntax and examples?

Reply:
Also, should this be called "Technical limitation" instead of "Technical limitations"?